examples/swr-site/pages/docs/custom-header-ids.es-ES.mdx

```diff
@@ -0,0 +1,15 @@
+# ID de Encabezado Personalizado
+
+Puede especificar identificadores de encabezado personalizados usando el formato, `## Header [#custom-id]`. Por ejemplo:
+
+```markdown
+## Foo [#bar]
+```
+
+## Esto Tiene una Identificación Personalizada [#ejemplo-id-personalizado]
+
+Puede ver la identificación del encabezado personalizado en el encabezado de arriba. Tenga en cuenta tanto al pasar el cursor sobre el encabezado como en la tabla de contenido, el href `ejemplo-id-personalizado` está en uso.
+
+## Esto No Tiene
+
+Este encabezado no tiene una identificación personalizada. El id se genera a partir del texto del encabezado.
```

Comment:
we don't need translation for this change, it's better to add some test for this change

Reply:
1. Do you want the non-English pages removed?
2. Could you give me more details on the sort of test you'd like? Is it an enhancement to an existing test, or something new and stand-alone? I'm unfamiliar with your test framework but I can give it a go with some guidance.

Reply:
you can remove non-english pages and about tests take a look at this file https://github.com/shuding/nextra/blob/main/packages/nextra/__test__/compile.test.ts

Reply:
I've added tests and updated snapshot files. Again, I'm unfamiliar with the process so let me know if I need to take a different approach.

Reply:
Looking into the linter errors...

Reply:
Done